notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    """
</pre><p>Is this docstring still needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>